changelog.md

```diff
@@ -1,3 +1,5 @@
+2024-11-08 - 479da8f1cb9f0f7edcfeee8232e1996d6b5d32f0 - PopOver.vue, Dialog.vue, PopOver.stories.js, pop-over.spec.ts - Modified PopOver component to be able to configure as persistent or not, Modified Dialog component to not to be able to close when ESC pressed while persistent is set to true
```

Comment:
Only mention the modified components. tests/stories are irrelevant to change log.

```md
2024-11-08 - 479da8f1cb9f0f7edcfeee8232e1996d6b5d32f0 - components/Dialog/Dialog.vue, components/PopOver/PopOver.vue - Modified PopOver component to be able to configure as persistent or not, Modified Dialog component to not to be able to close when ESC pressed while persistent is set to true
```

Reply:
I modified this...please check there are some other spec.ts files mentioned in the changelog.md